letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -444,8 +457,18 @@ if [ "$1" = "--le-auto-phase2" ]; then
 
     echo "Installing Python packages..."
     TEMP_DIR=$(TempDir)
+    trap "rm -rf $TEMP_DIR" EXIT
```

Comment:
Should be fine except if there's a space in $TEMP_DIR. I think we could put single quotes around $TEMP_DIR to fix it.